src/engine.cpp

```diff
@@ -247,43 +238,42 @@ bool Engine::loadNetwork() {
 
     // Storage for holding the input and output buffers
     // This will be passed to TensorRT for inference
-    m_buffers.resize(m_engine->getNbIOTensors());
+    //TODO: rollback libnvinfer API to TensorRT 8.2.1.9
+    //TODO: Needs a way to dynamically get input and output buffer size
+    int32_t nb_io_tensors = 2;
+    m_buffers.resize(nb_io_tensors);
+    // Tensor shape is (bs, chan, height, width)
+    nvinfer1::Dims4 inp_tensor_shape = {1, 3, 640, 640};
+    nvinfer1::Dims3 out_tensor_shape = {1, 84, 8400};
+    // Input and output tensor name
+    m_IOTensorNames.emplace_back("images");
+    m_IOTensorNames.emplace_back("output0");
```

Comment:
Can you look at v2.x of this project. Might give you some insight on how to dynamically get the input & output names and shapes, pre 8.6 

![image](https://github.com/cyrusbehr/tensorrt-cpp-api/assets/17056751/6b5f53d2-0629-4b92-a640-53918d64e57a)

Reply:
Thanks I will look into that, I was not sure actually how to get them dynamically as stated in the issue.
Another important issue is the INT8 compatibility.

Reply:
Ok. I'll merge in your changes for now so that others can reference it.